R/forestplot.R

```diff
@@ -11,19 +11,16 @@
 #' @export
 #'
 #' @examples
-#' mae <- hermes::multi_assay_experiment
-#' adtte <- teal.modules.hermes::rADTTE %>%
-#'   dplyr::mutate(is_event = (.data$CNSR == 0))
+#' data <- teal_data()
+#' data <- within(data, {
+#'   ADTTE <- teal.modules.hermes::rADTTE %>%
+#'     dplyr::mutate(is_event = .data$CNSR == 0)
+#'   MAE <- hermes::multi_assay_experiment
+#' })
+#' datanames <- c("ADTTE", "MAE")
+#' datanames(data) <- datanames
+#' join_keys(data) <- default_cdisc_join_keys[datanames]
```

Comment:
Please set keys manually for ADTTE - there is no ADSL in a data but the `default_cdisc_join_keys["ADTTE"]` returns also keys for `ADTTE -> ADSL`. Please fix in all examples and check the apps again

```suggestion
#' join_keys(data)["ADTTE", "ADTTE"] <- c("STUDYID", "USUBJID", "PARAMCD")
```

Reply:
Done, It was the case for `tm_g_km` and `tm_g_forest_tte` (and by extension `sample_tm_g_km` and `sample_tm_g_forest_tte`)